vertx-db2-client/src/main/java/examples/DB2ClientExamples.java

```diff
@@ -15,14 +15,13 @@
  */
 package examples;
 
-import java.math.BigDecimal;
 import java.util.Map;
 import java.util.stream.Collector;
 import java.util.stream.Collectors;
 
+import examples.Scratch.Days;
```

Comment:
looks like this import is breaking the build (since Scratch.java is not checked in)

Reply:
Yes, I missed that one.  Thanks for catching that for me!  I have corrected it and pushed out the fixed version.